dist/react-number-format.js

```diff
@@ -202,50 +202,8 @@ return /******/ (function(modules) { // webpackBootstrap
 	      }
 	    }
 	  }, {
-	    key: 'formatWithPattern',
```

Comment:
Looks like you deleted lots of stuff. I wonder if everything you removed really not needed :)